README.md

```diff
@@ -1,4 +1,92 @@
 # CDM Jupyterhub dockerfiles (Prototype)
 
 This prototype establishes a Docker container configuration for JupyterHub, designed to furnish a multi-user 
-environment tailored for executing Spark jobs via Jupyter notebooks.
\ No newline at end of file
+environment tailored for executing Spark jobs via Jupyter notebooks.
+
+## Using `docker-compose.yaml`
+
+To deploy the JupyterHub container and Spark nodes locally, execute the following command:
+
+```bash
+docker-compose up --build
+```
+
+## Test Submitting a Spark Job Locally
+
+### Submitting a Spark Job via spark-test-node
+```bash
+docker exec -it spark-test-node \
+    sh -c '
+	    /opt/bitnami/spark/bin/spark-submit \
+	    --master spark://spark-master:7077 \
```

Comment:
Can you make the spark master an environment variable in docker-compose?

Reply:
👍

Reply:
:+1: